src/triage/component/catwalk/__init__.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         self.predictor = predictor
         self.subsets = subsets
         self.replace = replace
-        self.protected_groups_generator = protected_groups_generator or ProtectedGroupsGeneratorNoOp()
+        self.protected_groups_generator = protected_groups_generator
```

Comment:
I'm not sure I understand removing the `NoOp()` version here -- what happens if no protected groups are specified in the config?

Reply:
The `Protected...NoOp` as all the `NoOp`s are created in the Experiment's constructor  : 

https://github.com/dssg/triage/blob/21d99a0cc587d33674b106e493da5d9efbe0e185/src/triage/experiments/base.py#L314-L329

There was no need to have that option in the other file

Reply:
Ah, got it -- so this was just redundant. That makes sense!